src/dune_rules/preprocessing.ml

```diff
@@ -703,13 +703,20 @@ let pp_one_module sctx ~lib_name ~scope ~preprocessor_deps
                         let dir =
                           Path.build (Super_context.context sctx).build_dir
                         in
+                        let original_src : Dep.Set.t =
+                          match Module.source m ~ml_kind with
+                          | None -> Dep.Set.empty
```

Comment:
How is `None` possible?

Reply:
Not sure.

Reply:
I guess I could code error here. I don't see any way this could be None.

Reply:
It should be `Option.value_exn` then.